packages/gatsby-theme-carbon/src/components/markdown/Markdown.module.scss

```diff
@@ -106,6 +106,21 @@ ul :global(.bx--list--unordered) {
   margin-left: 1ch;
 }
 
+:global(.bx--list--ordered.bx--list--nested) {
```

Comment:
@jnm2377 Do you mind maybe moving the comment up hear and making it a little more descriptive? I think this markdown style sheet is getting buck wild and I won't remember what happened in a month haha.